yarn.lock

```diff
@@ -23205,6 +23205,7 @@ fsevents@^1.2.7:
 "jquery@npm:3.5.1":
   version: 3.5.1
   resolution: "jquery@npm:3.5.1"
+  checksum: c50818c84837902a60938c85786feb612f346a9918f7285409d605449460790b3815ad8aaf5806f19e89f1a79b27d327802b421ab354fe213f3fd1801e3562e6
```

Comment:
Please don't commit yarn.lock changes that aren't a result of adding or modifying dependencies in your branch

Reply:
I spoke to @dannycoates about this a week or so ago and I was told to check in checksum changes whenever they came up. We should make a teamwide decision for consistency

Reply:
If there was a mismatch somewhere and we need to get back in sync it should be done one time in a standalone branch. I've seen checksum changes get added to a couple PRs, and right now on main when I `yarn install` it generates a diff.

Reply:
I'm fine with that, but I hope we can resolve this soon. It's annoying to have to switch branches and open up a new pr so frequently on installs